README.md

```diff
@@ -1,70 +1,42 @@
-# Getting Started with Create React App
+# Project Name
```

Comment:
- [ ] Please add the project name here instead of `Project Name` word in line one